packages/lodestar-params/package.json

```diff
@@ -49,6 +49,7 @@
     "mocha": "^6.2.2"
   },
   "dependencies": {
+    "@chainsafe/ssz": "^0.6.9",
```

Comment:
@wemeetagain Is this circular dep. Doesn't ssz depend on params?

Reply:
Maybe we should move this to lodestar-types?

Reply:
This shouldn't be a circular dependency, we will have:
- ssz -> params
- ssz -> types
  params -> types

The one tiny wrinkle is that we will have defined a few 'extra' ssz types, Number64, BigInt64, etc. As they are both needed here and in types. This is really really small, and likely not worth resolving fully. (or we could resolve it by creating these as 'well-known' types/singletons in the ssz package)

We can't move the `BeaconParams` ssz type to the types package because we want to parse the minimal/mainnet yaml here.

I think we can just treat this a a special type, we likely don't need to include it in our `IBeaconSSZTypes` interface, but just let it exist independently. It won't rely on the config to exist, and likely we can/should just let it exist as a singleton, here.